observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -5,3 +5,143 @@ include::_attributes/common-attributes.adoc[]
+
+Filters:: Transform or drop log messages in the pipeline. Users can define filters that match certain log fields and drop or modify the messages. Filters are applied in the order specified in the pipeline.
+
+== Configuring Inputs
+
+Inputs are configured in an array under `spec.inputs`. There are three built-in input types:
+
+application:: Selects logs from all application containers, except those in infrastructure namespaces like `openshift-*` and `kube-*`.
+
+infrastructure:: Selects logs from infrastructure components running in `openshift-*` and `kube-*` namespaces and node logs.
+
+audit:: Selects logs from the OpenShift API server audit logs, Kubernetes API server audit logs and node audit logs from auditd.
+
+Users can define custom inputs of type `application` that select logs from specific namespaces or using pod labels.
+
+The `inputDefaults` field sets global options for inputs like log-level filtering.
+
+== Configuring Outputs
+
+Outputs are configured in an array under `spec.outputs`. Each output must have a unique name and a type. Supported types include:
+
+elasticsearch:: Forwards logs to an external Elasticsearch instance.
+fluentdForward:: Forwards logs to an external Fluentd instance.
+syslog:: Forwards logs to an external syslog server.
+kafka:: Forwards logs to a Kafka broker.
+loki:: Forwards logs to a Loki logging backend.
+cloudwatch:: Forwards logs to AWS Cloudwatch.
+
+Each output type has its own configuration fields. Common ones include:
+
+url:: URL of the output destination.
+secret:: Name of an OpenShift secret containing authentication credentials for the output.
+tls:: TLS client configuration for the output.
+
+The `outputDefaults` field sets global options for outputs like buffer parameters.
+
+== Configuring Pipelines
+
+Pipelines are configured in an array under `spec.pipelines`. Each pipeline must have a unique name and consists of:
+
+inputRefs:: Names of inputs whose logs should be forwarded to this pipeline.
+outputRefs:: Names of outputs to send logs to.
+filterRefs:: (optional) Names of filters to apply.
+
+The order of filterRefs matters, as they are applied sequentially. Earlier filters can drop messages that will not be processed by later filters.
+
+== Configuring Filters
+
+Filters are configured in an array under `spec.filters`. They can match incoming log messages based on the value of structured fields and modify or drop them.
+
+Users can configure the following types of filters:
+
+labelFilter:: Adds labels to logs matching the filter. Useful for labeling certain types of logs.
+anonymizeFilter:: Masks user-sensitive fields using hash functions. Useful for GDPR compliance.
+nullFilter:: Drops logs matching the filter.
+
+Each filter type has a `match` configuration for matching fields and filter-specific parameters.
+
+== Enabling Output Security
+
+Outputs can be configured to use TLS/SSL by specifying a `secret` containing the TLS certificate and key. The secret can be created using:
```

Comment:
This is not totally correct. As it can be indicated the CA, tls certificate and key. More information in https://issues.redhat.com/browse/OBSDOCS-831

Also, it could be "Disabled Output Security` skipping the TLS verification. More information in (insecureSkipVerify)[https://issues.redhat.com/browse/OBSDOCS-773 and https://github.com/openshift/cluster-logging-operator/blob/782c47b9440d45e272e172f3ab07772cfd5fceb6/config/crd/bases/observability.openshift.io_clusterlogforwarders.yaml#L2337]